src/uproot/behaviors/TBranch.py

```diff
@@ -948,7 +948,7 @@ def show(
                 + "\n"
             )
 
-        for name, branch in self.iteritems(  # noqa: B301 (not a dict)
+        for name, branch in self.iteritems(
```

Comment:
These warnings have been removed from bugbear after it dropped Python 2 support.